certif/app/components/examiner-report-modal.js

```diff
@@ -0,0 +1,26 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class ExaminerReportModal extends Component {
+  @service store
+
+  reportOfTypeOther = this.args.report.examinerComment;
+
+  @tracked
+  isReportOfTypeOtherChecked = false
+
+  @action
+  async toggleShowReportOfTypeOther() {
+    this.isReportOfTypeOtherChecked = !this.isReportOfTypeOtherChecked;
+  }
+
+  @action
+  submitReport() {
+    if (this.reportOfTypeOther.length <= this.args.maxlength) {
+      this.args.report.examinerComment = this.reportOfTypeOther;
+    }
```

Comment:
y'a quelque chose dans le front qui empêche ça ? Sinon si le message est trop long, on fait rien et on ferme la modal en mode silencieux ? Pas de message d'erreur.

J'aurais plutôt vu : 
```
if (this.reportOfTypeOther.length <= this.args.maxlength) {
      this.args.report.examinerComment = this.reportOfTypeOther;
      this.args.closeModal();
} else {
    // montrer une erreur et ne pas fermer
}
```

Reply:
je réitère, je pense qu'on aura à terme un component par type de signalement vu qu'en plus ils vont avoir des logiques de validation différentes.

Reply:
Plus simplement, il faudrait que la saisie limite le nombre de caractere.

Reply:
sachant que c'est deja le cas via @maxLength